packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -164,6 +164,97 @@ const installNodeModulesTasks = ({ newAppDir }) => {
   ]
 }
 
+const displaySuccessMessage = () => {
+  const items = displaySuccessMessageItems
+
+  return Object.keys(items).forEach((key) => {
```

Comment:
Checkout `Object.entries`

Reply:
@peterp TIL, thank you 😄 
However, I will make an update to PR using `.map()` as you suggested below. More straight forward in this case.